docs/blob_shapes.rst

```diff
@@ -0,0 +1,32 @@
+.. _blob-shapes:
+   
+
+Blob Shapes
+===========
+
+We can choose between four different blob shapes that are specified with the ``blob_shape`` argument of the ``Model`` class.
+
+The blob shape consists of two parts, the blob shape in the propagation direction and the blob shape in the perpendicular direction thereof.
+The propagation direction is calculated from vx and vy of each individual blob (see :ref:`blob-alignment` for further details).
+
+The blob shape in the perpendicular direction is always set to "gauss", whereas you can choose one of the following blob shapes for the propagation direction:
```

Comment:
The blob shape in the perp direction can also be set to any shape, "gauss" is just the default.